<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ship</class>
 <widget class="QWidget" name="ship">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>199</green>
        <blue>123</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>199</green>
        <blue>123</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>236</red>
        <green>199</green>
        <blue>123</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="shipLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CONTROL</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="shipProgressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>760</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar{
	border : 2px rgb(201, 255, 236);
	border-radius: 3px;
}

QProgressBar::chunk {
	background-color:  rgb(165, 195, 200);
	width: 17px;
	margin:1px;
}</string>
   </property>
   <property name="maximum">
    <number>40</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="invertedAppearance">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="launchButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>430</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color: rgb(255, 99, 100); }</string>
   </property>
   <property name="text">
    <string>LAUNCH</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb0">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>320</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>350</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb7">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QCheckBox" name="cb8">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>350</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">cbGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="paintContainer">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>290</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="destLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>SET YOUR DESTINATION:</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>270</y>
     <width>141</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
	background-color: rgb(80, 90, 109)
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="shipPic">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>240</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/resources/shipImg.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>shipLabel</zorder>
  <zorder>shipProgressBar</zorder>
  <zorder>launchButton</zorder>
  <zorder>cb0</zorder>
  <zorder>cb1</zorder>
  <zorder>cb2</zorder>
  <zorder>cb3</zorder>
  <zorder>cb4</zorder>
  <zorder>cb5</zorder>
  <zorder>cb6</zorder>
  <zorder>cb7</zorder>
  <zorder>cb8</zorder>
  <zorder>paintContainer</zorder>
  <zorder>destLabel</zorder>
  <zorder>shipPic</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="cbGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
